docs/source/models/concept_models/vivarium_mncnh_portfolio/hemoglobin_module/module_document.rst

```diff
@@ -111,194 +149,190 @@ This module will:
+    - Use the :ref:`anemia intervention propensity <2024_vivarium_mncnh_portfolio_initial_attributes_module>` and the scenario-specific ferritin screening coverage value defined in the :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>` to determine if an individual simulant is screened for hemoglobin.
     - 
-  * - 9
-    - IV iron?
-    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. Probability of "yes" is equal to scenario-specific coverage.
+  * - 5
+    - Ferritin screening test result
+    - Use the instructions detailed on the :ref:`anemia screening intervention page <anemia_screening>`. Anemia status should be assessed according to first trimester hemolobin exposure output from the `First Trimester Hemoglobin Module`_.
     - 
-  * - 10
-    - Also receive IFA/MMS *for the first time* at late pregnancy visit?
-    - Coverage defined by scenario, see :ref:`pregnancy component scenario table <MNCNH pregnancy component scenario table>`. If answer to decision node #3 is no, then answer to this decision node is also no. Answer can only be yes if ANC attendance==ANC in later pregnancy ONLY. Then, probability of "yes" is equal to scenario-specific coverage.
-    - Use same propensity value as decision node #2 and #5 to answer this question
 
+.. list-table:: Anemia screening module outputs
+  :header-rows: 1
+
+  * - Output
+    - Value
+    - Dependencies
+  * - Hemoglobin screening result
+    - :code:`not_tested` / :code:`low` / :code:`adequate`
+    - V&V (via observation), simulation result
+  * - Ferritin screening result
+    - :code:`not_tested` / :code:`low` / :code:`adequate`
+    - V&V (via observation), simulation result
+  * - True hemoglobin exposure
+    - :code:`low` if first trimester hemoglobin <100 g/L, :code:`adequate if first trimester hemoglobin 100+ g/L`
+    - Used for V&V (via observation) to assess sensitivity and specificity of the hemoglobin screening test and as a convenient stratifying variable for specific observed outcomes. Not used as an input to any other module.
 
 
-2.4 Module Action Points
----------------------------
+End of Pregnancy Hemoglobin Module
+-------------------------------------------
 
-.. list-table:: Hemoglobin module action point
+This module applies the effect of the IV iron intervention for those who received it and applies the effect of oral iron intervention for those who have not already received the effect from their earlier first trimester ANC visit. Only those who have "low" test results for both the hemoglobin and ferritin screenings are eligible for IV iron. We assume that among those who receive both IV and oral iron interventions at the later pregnancy ANC visit, they receive only the effect of IV iron on their hemoglobin exposure rather than the additive impact of both interventions.
+
+.. image:: end_of_pregnancy_diagram.drawio.png
```

Comment:
It's pretty self-explanatory but can you add an arrow from box 2 "Eligible for IV iron" for "No"?